.rubocop.yml

```diff
@@ -37,4 +37,8 @@ Metrics/AbcSize:
   Max: 20
 
 Metrics/ModuleLength:
-  Max: 120
\ No newline at end of file
+  Max: 120
+
+Metrics/ClassLength:
+  Exclude:
+    - 'app/models/spotlight/resource.rb'
```

Comment:
Happy to discuss this exception.. I tried a couple different refactorings for `Spotlight::Resource` and none of them seemed to improve readability.